sympy/matrices/tests/test_reductions.py

```diff
@@ -1,11 +1,13 @@
+from sympy import symbols, I
 from sympy.matrices.common import _MinimalMatrix, _CastableMatrix
 from sympy.matrices.matrices import MatrixReductions
 from sympy.testing.pytest import raises
-from sympy.matrices import Matrix
+from sympy.matrices import Matrix, zeros
 from sympy.core.symbol import Symbol
 from sympy.core.numbers import Rational
 from sympy.functions.elementary.miscellaneous import sqrt
-from sympy.simplify.simplify import simplify
+#from sympy.simplify.simplify import simplify
```

Comment:
don't comment it is better to remove if it is not used.

Reply:
from the flake8 error It seems that it is required so please uncomment it.

Reply:
Sure. Working on it.